Readme.md

```diff
@@ -43,6 +43,9 @@ Visual Studio 2017 is a hard requirement. Other compilers might be
 able to build Dolphin on Windows but have not been tested and are not
 recommended to be used. Git and Windows 10 SDK 10.0.15063.0 must be installed when building.
 
+The "Release" solution configuration is designed for end users and includes performance
```

Comment:
Not too convinced about the wording here... Developers also have a legitimate reason to use release builds: it's much faster than debug builds, and sometimes issues only happen in release (optimised) builds.

Reply:
I'll reword this.